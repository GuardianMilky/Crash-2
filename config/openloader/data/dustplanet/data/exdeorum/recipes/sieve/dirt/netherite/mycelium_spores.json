{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:netherite_mesh","result":"exdeorum:mycelium_spores","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.2}}