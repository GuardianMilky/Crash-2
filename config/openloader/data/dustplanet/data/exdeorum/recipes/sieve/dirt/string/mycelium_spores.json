{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:string_mesh","result":"exdeorum:mycelium_spores","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.03}}