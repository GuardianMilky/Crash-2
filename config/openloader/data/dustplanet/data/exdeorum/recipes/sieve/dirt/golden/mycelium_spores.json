{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:golden_mesh","result":"exdeorum:mycelium_spores","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.13}}